frontend/src/typings/FormChange.ts

```diff
@@ -0,0 +1,4 @@
+export type FormChangePaylod = {
+  name: string;
+  value: unknown;
+};
```

Comment:
Not sure if this type already existed

Reply:
I think there is just `CheckInputChanged`.

Reply:
Seems to be made for checkboxes/switches  
It only accepts booleans

Reply:
Yeah, just pointing out that's the only one, though this typing should be moved to that file and probably renamed to something like `InputChanged` until we can be type all of them.